tests/full-example.ds

```diff
@@ -12,17 +12,15 @@ struct vec3 {
 
 impl vec3 {
   fn length(): f32 {
```

Comment:
just saying but both methods could be converted to the new syntax

Reply:
I'm demonstrating both types of return.